cookbooks/tile/templates/default/export.erb

```diff
@@ -54,6 +60,105 @@ def output_error(message, status = "400 Bad Request"):
   print("</body>")
   print("</html>")
 
+# Add a copyright notice for raster formats (PNG, JPEG, WEBP)
+def add_copyright_notice_raster(image, map_width, map_height, format):
+  # Convert the Mapnik image to PNG and store it in a BytesIO object
+  png = image.tostring("png")
+  png_io = io.BytesIO(png)
+
+  # Load the PNG data from the BytesIO object into a Cairo ImageSurface
+  surface = cairo.ImageSurface.create_from_png(png_io)
+
+  add_copyright_notice_vector(surface, map_width, map_height)
+
+  # Convert the Cairo surface to PNG in a BytesIO object
+  output_io = io.BytesIO()
+  surface.write_to_png(output_io)
+
+  if format == "png":
+    return output_io
+  else:
+    # Open the output PNG image for conversion to other formats
+    img = Image.open(output_io)
+    img_io = io.BytesIO()
+    img.save(img_io, format=format, quality=100)
```

Comment:
I don't think we want `quality=100` here... It seems like a really bad idea for [JPEG](https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html#jpeg-saving) and for [webp](https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html#webp-saving) is there a good reason not to go with the default? A better question for webp is whether to do lossy or lossless I suspect?

Reply:
I guess we can go with the default values. I will remove the quality parameter.